pkg/config/config.go

```diff
@@ -714,6 +714,7 @@ func InitConfig(config Config) {
 	config.SetKnown("process_config.expvar_port")
 	config.SetKnown("process_config.log_file")
 	config.SetKnown("process_config.profiling.enabled")
+	config.BindEnvAndSetDefault("process_config.remote_tagger", false)
```

Comment:
@juliogreff thanks for working on this. I believe process-agent for some historical reason doesn't play well with `BindEnvAndSetDefault`, we have a standalone function to load them instead, an example PR would be this: https://github.com/DataDog/datadog-agent/pull/7024/files#diff-69c5fcd6c4bca1aedf14eda11845d90134c99e08b99f99bcc07a9e984a8fa144R436, could you update your PR to follow the same code path? Thanks.

Reply:
PR updated, thanks for the heads up!